coordinator/src/main/scala/com/socrata/datacoordinator/common/util/DatasetIdNormalizer.scala

```diff
@@ -0,0 +1,26 @@
+package com.socrata.datacoordinator.common.util
+
+import com.ibm.icu.text.Normalizer
+import com.rojoma.json.v3.io.{IdentifierEvent, FieldEvent, StringEvent, JsonEvent}
+
+object DatasetIdNormalizer {
+
+  val normalizationMode: Normalizer.Mode = Normalizer.NFC
```

Comment:
Oh, and ever changing it would actually break things, as this controls the canonicalization things are sent to the database with.